<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SorterForm</class>
 <widget class="QWidget" name="SorterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Load file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Wind</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="windMaxDoubleSpinBox">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="windMinDoubleSpinBox">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="windRangeDoubleSpinBox">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Air temperature</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="tempMaxDoubleSpinBox">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="tempRangeDoubleSpinBox">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="tempMinDoubleSpinBox">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Save table</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Generate image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
